src/main/scala/fi/aalto/cs/apluscourses/intellij/actions/ReplAction.scala

```diff
@@ -27,7 +28,8 @@ class ReplAction extends RunConsoleAction {
 
   def getModuleWorkDir(@NotNull module: Module): String = {
     //  for project "root" this points to .../.../.idea folder
-    ModuleUtilCore.getModuleDirPath(module).replace("/.idea", "")
+    toSystemIndependentName(ModuleUtilCore.getModuleDirPath(module)
+      .replace("/.idea", ""))
```

Comment:
Should `replace()` be called for the system-independent name? I'm afraid `"/.idea"` does not match to any path in Windows because they don't have forward-slashes there. Or did I misunderstood?